Chapters/1-5-Chapter5.md

```diff
@@ -0,0 +1,155 @@
+
+"How can we get through them?" I asked, feeling doubtful.
+
+"Leave that to me. This is why Kipnu sent me after all. I'll distract them while you go around and slip through the door."
+
+A pang of anger welled up in me but I kept my eyes faced forward, "What sort of plan is that?"
+
+"Oh I think you'll find I can be very distracting."
+
+"Anyone can be distracting. You're a fidelant. Why not use that to get me in?"
+
+"The power of law lies in its enforcement. And trust me when I say these guards know enforcement. Not even I can wrest them from their duty to the law."
+
+"Noble. But annoying."
+
+I began to relish in Fajro's slow walking speed, imagining how I might slip by them unnoticed. It seemed impossible but for my shorter stature. There are some perks to being a maman.
+
+"What shall I do if they notice me?" I asked.
+
+"Keep going. You'll be imprisoned either way."
+
+"That's great to know."
+
+At about ten paces away from the guards one of them called out, "The Deliberation Hall is closed. Turn around."
+
+Fajro didn't stop and kept eye contact with them while they approached. I fell behind a bit, hiding behind him. _What was he going to do?_
+
+The guard called out again, "You two, turn around and go back where you came from!"
+
+Fajro responded to him confidently, "Tinpo, it's been a while, hasn't it."
+
+"I'm not telling you again. Turn back."
+
+"What sort of welcome is this? You don't recognize your uncle anymore?"
+
+"... Fajro? What are you doing here?" The guard called out and stopped.
+
+_The guard is his nephew?!_
+
+Fajro stopped in front of Tinpo and looked him up and down.
+
+"Tinpo, you make a fine guard," he said in his schmooziest voice.
+
+"Thank you uncle," he rested his stance and smiled a little bit.
+
+"How long have you been posted here? I haven't heard from your parents since I left Holrag."
+
+"They posted me near the railyards immediately after you referred me, but I've been working my way up here ever since."
+
+"Congratulations, my boy. It's only been a few ma'tova. Quite an achievement."
+
+"Thank you, sir."
+
+"And who's your friend over here?" he asked.
+
+"Oh, that's Padji. Thi was posted here around the same time I was."
+
+"You must be good friends. I want to meet hae."
+
+"Oh.. uh sure."
+
+We walked over to where Tinpo had been standing, and just before we got there, Fajro nudged me aside. At first I thought it was an accident because he made it seem that way, but then he stood opposite me and started talking to both of them, shifting their gaze away from the door, and away from me. I saw my moment and quickly tiptoed to the doorway. I lifted the heavy door a bit as I cracked it open: a trick my mother had taught me to stay quiet while we shared a room for sleeping. I slipped inside, and quietly closed it the same way, and then turned to face the most powerful governing body in the world.
```

Comment:
>  and then turned to face the most powerful governing body in the world.

Wait there are other national governments on this planet?

Reply:
Yea if there are city-states with borders, there are likely municipal governments. Is that important to get into?